content/docs/projects/csi-driver.md

```diff
@@ -1,42 +1,30 @@
 ---
 title: csi-driver
-description: ''
+description: 'Mounting cert-manager certificates without secrets'
 ---
 
-csi-driver is a Container Storage Interface (CSI) driver plugin for Kubernetes
-to work along cert-manager. The goal for this plugin is to seamlessly request
-and mount certificate key pairs to pods. This is useful for facilitating mTLS,
-or otherwise securing connections of pods with guaranteed present certificates
-whilst having all of the features that cert-manager provides.
-
-## Why a CSI Driver?
-
-- Ensure private keys never leave the node and are never sent over the network.
-  All private keys are stored locally on the node.
-- Unique key and certificate per application replica with a grantee to be
-  present on application run time.
-- Reduce resource management overhead by defining certificate request spec
-  in-line of the Kubernetes Pod template.
-- Automatic renewal of certificates based on expiry of each individual
-  certificate.
-- Keys and certificates are destroyed during application termination.
-- Scope for extending plugin behavior with visibility on each replica's
-  certificate request and termination.
-
-## Requirements and Installation
-
-This CSI driver plugin makes use of the 'CSI inline volume' feature - Alpha as
-of `v1.15` and beta in `v1.16`. Kubernetes versions `v1.16` and higher require
-no extra configuration however `v1.15` requires the following feature gate set:
-```
---feature-gates=CSIInlineVolume=true
-```
+csi-driver is a [Container Storage Interface (CSI)](https://kubernetes-csi.github.io/docs/) driver plugin for Kubernetes
+which works alongside cert-manager.
+
+Pods which mount the cert-manager csi-driver will request certificates from cert-manager
+without needing a `Certificate` resource to be created. These certificates will be mounted
+directly into the pod, with no intermediate Secret being created.
+
+## Why use csi-driver?
+
+- Ensure private keys never leave the node and are never sent over the network. All private keys are stored locally on the node
```

Comment:
I think we can say that these private keys do not hit the disk, they are kept in a memory-backed mount.